### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by 12Mosch** [Swedz/tesseract-neoforge#2]
[crash-2024-08-28_10.07.46-client.txt](https://github.com/user-attachments/files/16779182/crash-2024-08-28_10.07.46-client.txt)
[latest.log](https://github.com/user-attachments/files/16779247/latest.log)

```
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Constant modifier method getDefaultDyeColor(ILcom/llamalad7/mixinextras/sugar/ref/LocalRef;)I in tesseract_api.mixins.json:client.DynamicDyedItemDefaultDyeColorMixin from mod tesseract_api failed injection check, (0/1) succeeded. Scanned 0 target(s). No refMap loaded.
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:531)
```

**Comment by Swedz** [comment]
Fixed in 1.3.1

**Comment by 12Mosch** [comment]
thanks
